build/gen_stub.php

```diff
@@ -970,6 +983,10 @@ public function getMethodSynopsisFilename(): string {
         return implode('_', $this->name->parts);
     }
 
+    public function getAttributeName(): string {
```

Comment:
This name is a bit confusing. It's not the attribute name but the canonicalized (ie lowercased) function/method name to store the attach the given attribute to.

Reply:
Yes, you are right. However, I think it's better if the method name reflects the use-case (similarly to`getMethodSynopsisFilename()` and `getArgInfoName()`) instead of using  "canonicalized". That's why I changed `getAttributeName()` for `getNameForAttributes()` in https://github.com/php/php-src/commit/b2ed625de16304d8bd9f962beedb49b926e558b8. Hopefully, it addresses your comments and make intention more clear.

Reply:
Yes, perfect 🙂